### DISCUSSION TITLE
What is the difference between Mix.Cms.Api and Mix.Cms.Api.RestFul?

### DISCUSSION THREAD

**Discussion by DavidDury** [mixcore/mix.core#427] [Q&A] [answered]
Hi,

I am trying to understand what is the difference between Mix.Cms.Api and Mix.Cms.Api.RestFul?

Best Regards,
David

**Comment by nhathoang989** [comment, marked as answer]
Hi DavidDury.
Mixcore started off as a basic blog, therefore I didn't put much thought into the infrastructure.
It became too complicated after that, so I attempted to reorganize it as a Contentful CMS and Mix.Cms.Api.RestFul was born later.
But finally, We decided to rebuild the whole structure of Mixcore. You can take a look on the branch develop-v2.
